src/Donations/PaymentMethods/NewPaypal.tsx

```diff
@@ -65,10 +75,18 @@ function NewPaypal({
       payDonationFunction("paypal", "paypal", data);
     });
   }
-
+  function onApproveSubscription(data, actions) {
+    alert("You have successfully created subscription " + data.subscriptionID);
```

Comment:
Don't forget to remove this before merging:

<img width="446" alt="Bildschirmfoto 2021-12-27 um 13 33 04" src="https://user-images.githubusercontent.com/1532418/147472215-129fedf2-0db3-4a48-bebb-47f353849414.png">

Reply:
Yes. I removed an alert.